climada/util/plot.py

```diff
@@ -308,6 +308,7 @@ def geo_im_from_array(array_sub, coord, var_name, title,
     if 'vmax' not in kwargs:
         kwargs['vmax'] = np.nanmax(array_sub)
     if axes is None:
+        proj_plot = proj
```

Comment:
```suggestion
        proj_plot = proj
```
(see comment on lines 200, 201)